daemon/api.go

```diff
@@ -1173,6 +1174,7 @@ func postSnap(c *Command, r *http.Request, user *auth.UserState) Response {
 	if user != nil {
 		inst.userID = user.ID
 	}
+	inst.ctx = store.WithClientUserAgent(r.Context(), r)
```

Comment:
as we discussed the last time around, I think we should move the WithClientUserAgent to Command.ServeHTTP itself

Reply:
Indeed, I fixed this now.